edk2toolext/tests/test_edk2_setup.py

```diff
@@ -79,3 +79,41 @@ def test_setup_bad_omnicache_path(self):
         except SystemExit as e:
             self.assertEqual(e.code, 0, "We should have a non zero error code")
             pass
+
+    def test_parse_command_line_options_pass(self):
+        builder = Edk2PlatformSetup()
+        settings_file = os.path.join(self.minimalTree, "settings.py")
+        sys.argv = ["stuart_setup",
+                    "-c", settings_file,
+                    "BLD_*_VAR",
+                    "VAR",
+                    "BLD_DEBUG_VAR2",
+                    "BLD_RELEASE_VAR2",
+                    "TEST_VAR=TEST",
+                    "BLD_*_TEST_VAR2=TEST"]
+
+        try:
+            builder.Invoke()
+        except SystemExit as e:
+            self.assertEqual(e.code, 0)
+
+        env = shell_environment.GetBuildVars()
+        self.assertEqual(env.GetValue("BLD_*_VAR"), "TRUE")
+        self.assertEqual(env.GetValue("VAR"), "TRUE")
+        self.assertEqual(env.GetValue("BLD_DEBUG_VAR2"), "TRUE")
+        self.assertEqual(env.GetValue("BLD_RELEASE_VAR2"), "TRUE")
```

Comment:
these 4 should only be testing to make sure they are defined.  Not their value.

Reply:
Updated to check verify that it exists via `self.assertIsNotNone()`